tests/test_mets.py

```diff
@@ -378,5 +381,201 @@ def test_full_mets(self):
 
         mw.append_file(sip)
         mw.write('full_metsrw.xml', fully_qualified=True, pretty_print=True)
+        self.assert_mets_valid(mw.serialize())
```

Comment:
Yes, it fails because the METS file in this test is not really valid (according to the schematron file) because it contains fake metadata elements such as `<premis>agent</premis>`. The validation validates not only the METS but also the embedded PREMIS elements.